add info 23.1 (#186)
DevExpress/dotnet-eud: interface-elements-for-web/articles/document-viewer/viewing-and-navigating/search-for-a-specific-text.md

![EUD_HTML5DV_SearchPanel](../../../images/img121841.png)

> The **Search** panel allows you to enter the string to find and specify whether to use case-sensitive search and whether it is required to match the whole word during the search. To start search, press ENTER or click the **Search** button. Click **Stop** to stop the search process. You can use the **up** and **down** buttons to navigate through the search results in the document.

Author: ```suggestion
The **Search** panel allows you to enter a string to find and specify whether to use case-sensitive search and whether it is required to match the whole word during the search. To start the search, press ENTER or click the **Search** button. Click **Stop** to stop the search process. You can use the **up** and **down** buttons to navigate the document's search results.
```
